sqlmesh/dbt/manifest.py

```diff
@@ -270,6 +267,35 @@ def _load_profile(self) -> Profile:
             target_override=self.target.name,
         )
 
+    def _disabled_ref(self, ref: str) -> bool:
```

Comment:
[Nit] `is_ref_disabled`. This name looks like it returns a ref

Reply:
I secretly want a question mark in the method name